### DISCUSSION TITLE
CLA of wrong project?

### DISCUSSION THREAD

**Discussion by SebastianOpriel** [eclipse-edc/Connector#4691] [General]
We can find [here](https://github.com/eclipse-edc/Connector/blob/4eca6753b6de63a6664caa850f3b0d7dbb9f721d/README.md?plain=1#L59) a link to the Contributing guidelines. The document then refers to CLA of Google. I assume this might be wrong, but maybe I am missing something here.

**Comment by ndr-brt** [comment]
Is this referring to this repo right? https://github.com/eclipse-edc/eclipse-edc.github.io/blob/main/CONTRIBUTING.md

It looks to be a leftover from the Docsy scaffold. Feel free to open an issue on the [documentation repository.](https://github.com/eclipse-edc/eclipse-edc.github.io)

**Comment by SebastianOpriel** [comment]
Yes, you are right - tanks for the hint. Ok, create the issue [here](https://github.com/eclipse-edc/eclipse-edc.github.io/issues/49).
